pkg/config/config.go

```diff
@@ -1,69 +1,95 @@
 package config
 
 import (
-	"errors"
 	"fmt"
 	"io"
 
 	"gopkg.in/yaml.v2"
 )
 
-type Config struct {
-	PRFlow       string        `yaml:"prflow"`
-	Environments []Environment `yaml:"environments"`
+type PRFlowType string
+
+const (
+	PRFlowTypePerApp PRFlowType = "per-app"
+	PRFlowTypePerEnv PRFlowType = "per-env"
+)
+
+type App struct {
+	FeatureLabelSelector map[string]string `yaml:"featureLabelSelector"`
+}
+
+type Group struct {
+	Applications map[string]App `yaml:"applications"`
```

Comment:
my issue is that most people use the group apps. Causing confusion, which is why i think it is better to go with applications.

Reply:
I see what you mean:
```
    groups:
      apps:
        apps:
          podinfo:
            featureLabelSelector:
              app: podinfo
```
This is extra unfortunate because among our current customers, I don't think anyone should be using a group called "apps". Rather they should use narrower naming for their groups. Oh well.